src/bin/tectonic.rs

```diff
@@ -189,10 +190,11 @@ fn inner(
                     "something bad happened inside {}; its output follows:\n",
                     engine
                 );
-                tt_error_styled!(status, "===============================================================================");
+
+                tt_error!(status, "===============================================================================");
```

Comment:
Am I right that this changes the output in these situations to add an `error: ` prefix to the equals-sign lines?